ledger/participant-integration-api/src/main/scala/platform/apiserver/RateLimitingInterceptor.scala

```diff
@@ -75,6 +93,52 @@ private[apiserver] final class RateLimitingInterceptor(metrics: Metrics, config:
     }
   }
 
+  private def memoryOverloaded(fullMethodName: String): Either[String, Unit] = {
+    tenuredMemoryPool.fold[Either[String, Unit]](Right(())) { p =>
```

Comment:
How about abstracting this method and moving it to `TenuredMemoryPool`? It could just return simplified string to be further augmented with the details of `fullMethodName` outside